lib/screens/forgot_password/cubit/forgot_password_cubit.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:bloc/bloc.dart';
+import 'package:ecellapp/core/res/errors.dart';
+import 'package:ecellapp/screens/forgot_password/forgot_password_repository.dart';
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+import '../../../core/utils/logger.dart';
+import 'package:ecellapp/core/res/strings.dart';
+
+part 'forgot_password_state.dart';
+
+class ForgotPasswordCubit extends Cubit<ForgotPasswordState> {
+  final ForgotPasswordRepository _forgotPasswordRepository;
+
+  ForgotPasswordCubit(this._forgotPasswordRepository) : super(ForgotEmailInitial());
+
+  void sendOTP(String email, ForgotPasswordState state) async {
+    try {
+      emit(ForgotLoading());
+      await _forgotPasswordRepository.sendOTP(email);
+      emit(ForgotOTPInitial());
+    } on NetworkException {
+      emit(ForgotNetworkError(S.networkException, state));
+    } on ResponseException catch (e) {
+      emit(ForgotPasswordError(e.message, state));
```

Comment:
Technically, there is no need for ForgotNetworkError
```suggestion
    } on NetworkException {
      emit(ForgotPasswordError(S.networkException, state));
    } on ResponseException catch (e) {
      emit(ForgotPasswordError(e.message, state));
```
1. treat ForgotNetworkError same as ForgotPasswordError
2. remove ForgotNetworkError state
3. do the above changes in other functions too.